### PR TITLE
doc(README): add Shubham as a contributor

### PR DESCRIPTION
**Pull request by SHUBHAMDAVE001**
Fixes #<issue no>
Screenshot: (if applicable):  
Description of the change:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Wallora webapp recreated using React
 - Rajakvk (@rajakvk)
 - Kratika Chowdhary (@kratika0907)
 - KartheeK Paturu (@kartheekwd)
+- Shubham Dave (@SHUBHAMDAVE001)
```
Comment (souvikbasu) [inline README.md:16]: set GitHub username in small letters

**Review by souvikbasu — CHANGES_REQUESTED** [review]
set commit message in the format specified as-
doc(README): add Shubham as a contributor

File: README.md (hunk 1 above)
Comment (SHUBHAMDAVE001) [reply README.md:16]: Okay sure.

**Review by souvikbasu — CHANGES_REQUESTED** [review]
please fix commit message as discussed before

**Pull request closed without merging**